config/power_virtual_servers_regions.yml

```
@@ -0,0 +1,40 @@
+us-south:
+  :name: us-south
+  :description: Dallas, TX United States (deprecated)
+  :hostname: us-south.power-iaas.cloud.ibm.com
+eu-de:
+  :name: eu-de
+  :description: Frankfurt, Germany
+  :hostname: eu-de.power-iaas.cloud.ibm.com
+lon:
+  :name: London
+  :description: London, United Kingdom
+  :hostname: lon.power-iaas.cloud.ibm.com
+mon:
+  :name: Montreal
+  :description: Montreal, Canada
+  :hostname: mon.power-iaas.cloud.ibm.com
+osa:
+  :name: Osaka
+  :description: Osaka, Japan
+  :hostname: osa.power-iaas.cloud.ibm.com
+sao:
+  :name: Sao Paulo
+  :description: Sao Paulo, Brazil
+  :hostname: sao.power-iaas.cloud.ibm.com
+syd:
+  :name: Sydney
+  :description: Sydney, Austrailia
+  :hostname: syd.power-iaas.cloud.ibm.com
+tok:
+  :name: Tokyo
+  :description: Tokyo, Japan
+  :hostname: tok.power-iaas.cloud.ibm.com
+tor:
+  :name: Toronto
+  :description: Toronto, Canada
+  :hostname: tor.power-iaas.cloud.ibm.com
+wdc:
+  :name: Washington DC
+  :description: Washington DC, United States
+  :hostname: wdc.power-iaas.cloud.ibm.com
```

Need to add an a "Washington DC (us-east)" entry for `us-east` location:
```suggestion
  :hostname: wdc.power-iaas.cloud.ibm.com
:us-east
  :name: us-east
  :description: Washington DC, United States
  :hostname: us-east.power-iaas.cloud.ibm.com
```